FRINXio/frinx-frontend
# add transaction log; add revert changes feature

You need to test it with https://github.com/FRINXio/frinx-inventory-server/pull/136
Steps needed for this to work:
- make sure you have latest `master` in ui-dev-proxy
- in ui-dev-proxy set `INVENTORY_API_URL` based on the new `.env.example` value (it should end with `/graphql`)
- run `npm run start:uc`
- edit `.env` file in `frinx-frontend-server`: `INVENTORY_API_URL="/api/inventory"`
- test this against .7 machine
- start inventory server (make sure to use the branch from PR above)
- start frontend (`yarn run dev`)
